api/database/migrations/2023_12_12_204200_create_orders_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('store_id');
+            $table->unsignedBigInteger('store_id');
+            $table->unsignedBigInteger('category_id');
+            $table->integer('job_order')->nullable();
+            $table->integer('order_number');
+            $table->string('brand');
+            $table->string('brand');
+            $table->string('brand');
+            $table->decimal('downpayment')->default(0.00);
+            $table->integer('quantity')->default(1);
+            $table->enum('status', ['delivered', 'open', 'in-transit', 'cancelled', 'onhold'])->default('open');
+            $table->string('link')->nullable();
+            $table->string('notes')->nullable();
```

Comment:
since comment can be long. `string `is not enough so lets use `text`
```suggestion
            $table->text('notes')->nullable();
```